pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -219,6 +208,7 @@ protected MutableLong initialValue() throws Exception {
             return Collections.newSetFromMap(new IdentityHashMap<>());
         }
     };
+    io.netty.util.internal.InternalThreadLocalMap threadLocalMap = io.netty.util.internal.InternalThreadLocalMap.get();
```

Comment:
What's the purpose of adding this field

Reply:
This is used in `handlePulishBufferLimit: cnxsPerThread.get(threadLocalMap)`  Listener is trigged in another thread, and cnxsPerThread is a FastThreadLocal

Reply:
I got it